node/src/components/consensus/era_supervisor.rs

```diff
@@ -103,8 +103,6 @@ pub struct EraSupervisor<I> {
     next_executed_height: u64,
     #[data_size(skip)]
     metrics: ConsensusMetrics,
-    // TODO: discuss this quick fix
-    finished_joining: bool,
```

Comment:
We should be able to remove the `enqueued_events` below as well now.

Reply:
Actually, I wonder if we can... I'll do it and run some tests, but I think they might have to remain - it is still possible that some components will emit events to consensus before it is initialized, and it might cause problems.

Reply:
Per the conversation on Slack, it should be safe - in the validator state, the blocks can only be added when consensus finalizes them, and finalizing blocks means that it is already initialized, so `enqueued_events` is unnecessary.